### PR TITLE
Report updates

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

#1765 

## Background 
Enhanced reporting is essential to effectively showcase the unique selling proposition of asset tokenization on the blockchain. As a first step we want to improve the individual reporting tab of a pool.

## Requirements
- [x] Decrease the standard font size in the UI to increase density of the table 
- [x] Display currency amounts and token prices with 5 decimals
- [ ] ~~Rename type of first investment from `Investment order updated` to `Investment order submitted` 
Note that this will be a bit tricky to implement. The UI will need to figure out whether the outstanding order was 0 at the time of the new update order, to determine whether to use `investment order submitted` or `investment order updated`~~
- [x] Replace the date range with a date picker, from 03/21/2024-03/25/2024
- [x] Change name of export csv
    * Current name:
        * 2909679805-investor-transactions-Wed Nov 08 2023 10_23_02 GMT+0100 (Central European Standard Time)-Fri Dec 08 2023 10_23_02 GMT+0100 (Central European Standard Time)
         * New name:
2909679805-investor-transactions-Wed Nov 08 2023-Fri Dec 08 2023 
- [x] Current csv data is not ususable. Fix export data format and add extra column for the currency
    - `100,000 USDT` => `100000` + `USDT`
- [x] Rename “Type” --> “Transaction type” for all reports
- [x] Rename “USDC amount” --> “Currency amount” for all reports
- [x] Right align all numbers in the currency amount column
- [x] Link transaction hash to investor and borrower transactions. Add a external link icon at the end of the row and link to the Centrifuge Chain transaction on Subscan
- [x] Add filters to `Network` and `Type`
- [x] Add filter to sort from small to large/low to high `Date`, `Token amount`, `Price`

### NEW Token price report spin off from pool balance report
- [x] Create new report `Token price`
- [x] Selector `Tranche` that displays all tranches and selector `Date range` with date picker and selector `Group` by `Day` and `Month`
- [x] Two rows: Token price and Token supply

### Change pool balance report
- [x] Add these 4 rows and balance and keep investments and redemptions
- `Asset value`
- Reserve --> `Onchain reserve` (copy change)
- `Cash accounts ` (skipped, blocked by subquery)
- `Pool value`
- add a "Change" row for Pool Value (i.e. "+10,000 / +0.5%") to show day to day change in pool value/NAV

### Investor transactions 
- [x] Add onchain transaction link at the end of the row to track each transaction
- [x] Add new selector `Address` and search field in the selector dropdown to search for an address 

### Asset transactions
- [x] Show an additional selector “Asset” to show all transactions per asset 
- [x] Show an additional selector “Transaction type” and option to filter for Created, Financed or Repaid
- [x] Display asset name as the second column after asset ID
- [ ] ~~Copy change for pools that only have cash assets and oracle valuation for their assets, Financed --> Purchase, Repaid ---> Sale (if possible)~~
- [x] Remove abbreviation with “k” and display full number
image

### Asset list
- [x] Remove date range selector and add selector for Status `Active`, `Repaid`, `Overdue`

### Approvals

- [x] Dev

### CODE REVIEW

File: centrifuge-app/src/components/Report/Holders.tsx [hunk 1]
```diff
@@ -1,57 +1,93 @@
...
-  const { activeTranche, setCsvData } = React.useContext(ReportContext)
+  const { activeTranche, setCsvData, network, address } = React.useContext(ReportContext)
 
   const utils = useCentrifugeUtils()
   const holders = useHolders(pool.id, activeTranche === 'all' ? undefined : activeTranche)
 
+  const cellFormatters = [
+    noop,
+    copyable,
+    (v: any, row: any) => (typeof v === 'number' ? formatBalance(v, row[3], 5) : '-'),
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/Report/Holders.tsx:40]: any chance the typings could be improved here? Just looking at `row[3]` thinking what the heck is that? 😅

File: centrifuge-js/src/modules/pools.ts [hunk 2]
```diff
@@ -2591,7 +2579,7 @@ export function getPoolsModule(inst: Centrifuge) {
     return combineLatest([$query, getPoolCurrency([poolId])]).pipe(
       switchMap(([queryData, currency]) => {
         const currencyDecimals = currency.decimals
-
+        console.log('queryData?.investorTransactions.nodes', queryData?.investorTransactions.nodes)
```
Comment (sophialittlejohn) [inline centrifuge-js/src/modules/pools.ts:2582]: left over log

File: centrifuge-app/src/components/Report/InvestorTransactions.tsx [hunk 3]
```diff
@@ -1,143 +1,211 @@
...
+  'left',
+  'right',
+  'left',
+  'right',
+  'left',
+  'right',
+  'left',
+  'left',
+]
+const sortable = [false, false, false, false, true, false, true, false, true, false, true, false, false]
```
[the remark this answers is not in the document]
Comment (onnovisser) [reply centrifuge-app/src/components/Report/InvestorTransactions.tsx:50]: yes, it's messy and became very unwieldy with these larger tables. gonna refactor

[the remark this answers is not in the document]
Comment (onnovisser) [reply centrifuge-app/src/components/Report/InvestorTransactions.tsx]: @sophialittlejohn refactored

**Review by sophialittlejohn — APPROVED** [review]

File: centrifuge-app/src/components/Report/InvestorTransactions.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (sophialittlejohn) [reply centrifuge-app/src/components/Report/InvestorTransactions.tsx]: thanks! Defs a bit longer now but I think it's still easier to read

**Pull request merged**